contracts/Targeted/RIFGovernorTargeted.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.9;
+
+import '@openzeppelin/contracts/governance/Governor.sol';
+import './RIFProposalTarget.sol';
+
+abstract contract RIFGovernorTargeted is Governor {
```

Comment:
:+1: good refactor!

Reply:
Note that this is not specific to the governance FT, so can remove `RIF` from the name here as well.